docs/reference/pages/model_recipes.rst

```diff
@@ -196,3 +196,31 @@ Here, ``blogs.filter(tags__name=tag)`` invokes a reverse Django QuerySet filter
 Iterating through ``page.tags.all`` will display each tag associated with ``page``, while the link(s) back to the index make use of the filter option added to the ``BlogIndexPage`` model. A Django query could also use the ``tagged_items`` related name field to get ``BlogPage`` objects associated with a tag.
 
 This is just one possible way of creating a taxonomy for Wagtail objects. With all of the components for a taxonomy available through Wagtail, you should be able to fulfil even the most exotic taxonomic schemes.
+
+
+Have redirects created automatically when changing page slug
+------------------------------------------------------------
+
+You may want redirects created automatically when a url gets changed in the admin so as to avoid broken links. You can add something like the following block to `/models/wagtail_hooks.py`.
```

Comment:
`/models/wagtail_hooks.py` would only be a correct path if you had an app named "models" - I'd suggest saying "You can add something like the following block to a `wagtail_hooks.py` within one of your project's apps". (Also, note that the filename should be in double backticks rather than single - an annoying RestructuredText quirk there...)

Reply:
Good point. Have changed that.